string name
---
WorldPlace match
bool found
